CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to the Virtual Rainforest
+
+We're really happy that you are thinking about contributing to the Virtual Rainforest
+project. The whole point of the project is to generate a tool that can be used widely by
+the community and the best way for that to happen is for the community to build it.
+
+To make contributing as seemless as possible, please note these developer guidelines.
+
+## Development setup
+
+Further notes are available [here](source/docs/develop/developer_setup.md) but - once
+you have cloned the Virtual Rainforest repository to your own machine - you will need to
+set up our development toolchain on your machine to contribute to the project.
+
+* We use `poetry` ([](https://https://python-poetry.org/)) to manage the package
```

Comment:
You *could* make this an inline hyperlink:
```suggestion
* We use ([poetry](https://https://python-poetry.org/)) to manage the package
```

Same for other links. This is a stylistic choice though so up to you

Reply:
I guess I think it is useful for code tools to be styled - it tells people they are programs - and having a link on a code styled word can be less visible. Testing [`here`](https://httpstat.us/200).

Reply:
I hear you. As I say, up to you!

Reply:
That's not too bad as rendered in GitHub.